setup.py

```diff
@@ -51,8 +51,8 @@
         "eth-abi>=1.2.2,<2",
         "eth-utils>=1,<2",
         "ethpm>=0.1.4a1,<1",
+        "twig>=0.1.0a5,<1",
```

Comment:
This feels wrong.  `pytest-ethereum` is the low level library so I would expect it to know nothing about `twig` and instead for `twig` to install `pytest-ethereum` as a dependency.

Reply:
I was under the impression that all compilation was going to happen in `twig`? In which case, `pytest-ethereum` would need access to the compilation utils from `twig` in order to create and feed the manifest into `pytest-ethereum`'s deployers. Or am I mistaken, and vyper contract compilation should happen in `pytest-ethereum`?

Reply:
@pipermerriam We forgot to talk about this pr (and https://github.com/ethereum/twig/pull/11) in our 1-on-1. Still want to schedule a call to discuss?

Reply:
I realize we still haven't resolved this.  Ping me and we can chat.